app/views/pages/main.html

```diff
@@ -29,7 +29,8 @@
 
   <div class="waiter text-center" ng-if="isLoading.application || isLoading.package">
     <i class="fa fa-spinner fa-pulse fa-4x"></i>
-    <span>Loading...</span>
+    <span ng-if="false">Loading...</span>
```

Comment:
Because first span will be displayed while while js bundle is loading ( == app is not running, so i18n service is not available yet). And the second span will be displayed after app start.